frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react';
+import Header from '../../components/Header/Header';
+import Footer from '../../components/Footer/Footer';
+import TagsBar from './TagsBar/TagsBar';
+import recipephoto from '../../assets/bread.jpg';
+import { withRouter } from 'react-router-dom';
+import './Recipe.css';
+
+class Recipe extends Component {
+  constructor() {
+    super();
+    this.state = {
+     
```

Comment:
I think I'm going to get a reputation as white space police, but this one is probably not useful. :)